doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -143,12 +139,27 @@ list of assertion commands is given in :ref:`Assertions`. The command
 
    .. seealso:: :cmd:`Proof with`
 
-.. cmd:: Proof using {+ @ident }
+.. cmd:: Proof using @section_subset_expr {? with @ltac_expr }
+
+   .. insertprodn section_subset_expr starred_ident_ref
+
+   .. prodn::
+      section_subset_expr ::= {* @starred_ident_ref }
+      | {? - } @section_subset_expr50
+      section_subset_expr50 ::= @section_subset_expr0 - @section_subset_expr0
+      | @section_subset_expr0 + @section_subset_expr0
+      | @section_subset_expr0
+      section_subset_expr0 ::= @starred_ident_ref
+      | ( {* @starred_ident_ref } ) {? * }
+      | ( {? - } @section_subset_expr50 ) {? * }
+      starred_ident_ref ::= @ident {? * }
```

Comment:
Why not collapse the two rules into one by reusing the `section_subset_expr` nonterminal? It would also read more easily.